test-suite/output/Arguments.out

```diff
@@ -1,99 +1,75 @@
 Nat.sub : nat -> nat -> nat
 
 Nat.sub is not universe polymorphic
-Argument scopes are [nat_scope nat_scope]
-The reduction tactics unfold Nat.sub but avoid exposing match constructs
+Arguments Nat.sub _%nat_scope _%nat_scope : simpl nomatch
 Nat.sub is transparent
 Expands to: Constant Coq.Init.Nat.sub
 Nat.sub : nat -> nat -> nat
 
 Nat.sub is not universe polymorphic
-Argument scopes are [nat_scope nat_scope]
-The reduction tactics unfold Nat.sub when applied to 1 argument
-  but avoid exposing match constructs
+Arguments Nat.sub _%nat_scope / _%nat_scope : simpl nomatch
 Nat.sub is transparent
 Expands to: Constant Coq.Init.Nat.sub
 Nat.sub : nat -> nat -> nat
 
 Nat.sub is not universe polymorphic
-Argument scopes are [nat_scope nat_scope]
-The reduction tactics unfold Nat.sub
-  when the 1st argument evaluates to a constructor and
-  when applied to 1 argument but avoid exposing match constructs
```

Comment:
On the other hand the question of whether `Print` is a command that targets humans or machines does apply here. I find the previous output much better for most human users.

Reply:
I wonder if it might be worth keeping the explanations in addition to the arguments line?  They serve as inline documentation for the notation above (that documentation can be useful for beginners, and even non-beginners might not know all of the features of the `Arguments` command).

Reply:
I agree, except for the description of the scopes maybe, which was already hard to understand.